repositories/dbCommentRepository.js

```diff
@@ -0,0 +1,67 @@
+const { Comment } = require('../models');
+
+class DBCommentRepository {
+  async countComments() {
+    return await Comment.count();
+  }
+
+  async checkIfAnyCommentsExist() {
+    return (await this.countComments()) > 0;
+  }
+
+  async getCommentsByMovieId(movieId, page, limit) {
+    try {
+      const offset = (page - 1) * limit;
+      
+      const { count: totalCount, rows: comments } = await Comment.findAndCountAll({
+        where: { movie_id: movieId },
+        limit,
+        offset,
+        raw: true,
+      });
+      
+      const totalPages = Math.ceil(totalCount / limit);
+      
+      return {
+        response: comments,
+        pagination: {
+          totalPages,
+          currentPage: page,
+          hasNextPage: page < totalPages,
+          hasPreviousPage: page > 1,
+        },
+      };
+    } catch (err) {
+      console.error("Error getting comments", err.stack);
```

Comment:
You are still not re-raising these errors after logging. This is dangerous

Reply:
I am working on that now